### PR TITLE
Fix cmake library path when installing python package

### PR DESCRIPTION
**Pull request by larroy**
No code changes.

### CODE REVIEW

File: python/mxnet/libinfo.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def find_lib_path():
     """
     curr_path = os.path.dirname(os.path.abspath(os.path.expanduser(__file__)))
     api_path = os.path.join(curr_path, '../../lib/')
-    cmake_build_path = os.path.join(curr_path, '../../build/Release/')
+    cmake_build_path = os.path.join(curr_path, '../../build/')
```
Comment (piiswrong) [inline python/mxnet/libinfo.py]: why?

Comment (larroy) [reply python/mxnet/libinfo.py]: cmake artifact is put in build/ not in build/Release  Why is the previous path there I wonder...

Comment (cjolivier01) [reply python/mxnet/libinfo.py]: CMake often tends to make a separate output directory for each build configuration "Debug", "Release", "RelWithDebInfo", etc. (when tiolkd to do so, for example by CLion, except it calls it cmake-build-debug, cmake-build-release, etc.). While I am not crazy about trying to guess the cmake output directory, I think explicitly pointing to "Release" when what you think you're running is "Debug" is dangerous.

Comment (larroy) [reply python/mxnet/libinfo.py]: I never saw mxnet being left in this folder, did you observe it? Maybe with a different CMake version or intellij? Agreed with your remarks.

Comment (cjolivier01) [reply python/mxnet/libinfo.py]: Someone would need to do it manually when running cmake, but it's sort of a common practice.

Comment (sxjscience) [reply python/mxnet/libinfo.py]: I'm using Visual Studio in Windows and after building the solution file the dll will be in `build/Release`. I think this path should be added back.

Comment (sxjscience) [reply python/mxnet/libinfo.py]: OK, forget about the previous comment. The release option is added in the codes below.

**Pull request merged**